src/groundStation/groundStation.py

```diff
@@ -82,8 +84,27 @@ def __zmq__(self, addr):
 
     def __uart__(self, device):
         """ initialize uart interface """
-        libcsp.kiss_init(device, 115200, 512, 'uart')
+        ser = serial.Serial('/dev/ttyUSB1',                                      
```

Comment:
The path should be passed as a parameter. I believe the argparser has this parameter already

Reply:
I don't know how to get this - not good enough in Python but will ask Ryan/John

Reply:
I'm pretty sure the device parameter stores the string you need. Try printing it

Reply:
Yep that worked thanks